### PR TITLE
9 fetch zkappstateuseraccountstate

### LINKED ISSUE #9: Fetch zkAppState/UserAccountState
Scripts to get zkApp state or user account state from MINA nodes

### PR DESCRIPTION
**Pull request by hgedia**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ proofs on blockchain VMs. Users are recommended to check the proof market toolch
 
 - [Boost](https://www.boost.org/) == 1.76.0
 - [cmake](https://cmake.org/) >= 3.5
-- [clang](https://clang.llvm.org/) >= 14.0.6
+- [clang](https://clang.llvm.org/) == 12.0.0
```
Comment (SK0M0R0H) [inline README.md]: Do we really need anything related to c++ in this repo?

File: scripts/protocol/mina/get_mina_state.py [hunk 2]
```diff
@@ -63,16 +64,58 @@ def state_query_to_graphql(output_path, url):
...
-                    prog = 'Mina Helper for =nil; Proof Market',
-                    description = 'Mina Helper retrieves data related to Proof Market functionality from Mina node')
-    parser.add_argument('--url', help="GraphQL URL", default="http://localhost:3085/graphql")
-    parser.add_argument('--output', help="Output file path", default="proof_market_data.json")
+        prog='Mina Helper for =nil; Proof Market',
+        description='Mina Helper retrieves data related to Proof Market functionality from Mina node')
+    parser.add_argument('--url', help="GraphQL URL", default="https://proxy.berkeley.minaexplorer.com/")
+    parser.add_argument('--output', help="Output file path", default="output.json")
+    parser.add_argument('--address', help="Mina public key of zkApp or user", default="")
+    parser.add_argument('--type', help="Query type: ledger or account", default="")
```
Comment (SK0M0R0H) [inline scripts/protocol/mina/get_mina_state.py]: I think, it's better to do it via subparsers. 
Example:
https://github.com/NilFoundation/proof-market-toolchain/blob/master/scripts/ask_tools.py#L56

**Review by SK0M0R0H — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (hgedia) [reply README.md]: Agreed , removed.

File: scripts/protocol/mina/get_mina_state.py (hunk 2 above)
Comment (hgedia) [reply scripts/protocol/mina/get_mina_state.py]: Updated

**Review by SK0M0R0H — APPROVED** [review]

**Pull request merged**